src/librustdoc/html/static/main.js

```diff
@@ -696,39 +703,52 @@
             var search_input = document.getElementsByClassName('search-input')[0];
             search_input.onkeydown = null;
             search_input.onkeydown = function(e) {
-                var actives = [];
+                var actives = [[], [], []];
+                var current = 0;
```

Comment:
What does this `current` store? Is it the same as the `currentTab` from the outer scope?

Reply:
No, it's a local one when we're iterating over tabs.